examples/tests/invalid/parameters.test.w

```diff
@@ -0,0 +1,18 @@
+let app = nodeof(this).app;
+
+let registrar = app.parameterRegistrar;
+
+registrar.addParameterSchema({
+  type: "object",
+  properties: {
+    foo: {
+      type: "string"
+    }
+  },
+  required: ["foo"]
+});
```

Comment:
Of course:

```suggestion
struct MyParams {
  foo: str;
}

registrar.addParameterSchema(MyParams.schema());
```

Reply:
oh but of course!

Reply:
This is actually pretty fucking cool, I added a short video to the PR description @eladb, feels like the puzzle pieces are starting to make a picture